src/test/python/MCSVerification.py

```diff
@@ -63,38 +64,32 @@ def _run_scispark_implementation():
     Purpose: To run GTG runner for SciSpark results to compare as a spark-submit task
     Assumptions: spark-submit is available on the path. scala-2.11 is used.
     '''
-    runSpark = False
-
     print 'Checking on SciSpark implementation data' 
     of.write('\n Checking on SciSpark implementation data')
 
     try:
-        if not os.path.exists(workingDir + '/scisparkGTG'):
+        if os.path.exists(workingDir + '/scisparkGTG'):
+            shutil.rmtree(workingDir + '/scisparkGTG')
```

Comment:
@kwhitehall 
So it looks like if the directory already exists, then you remove it.
However, it doesn't look like it recreates the directory, after removing.

These mkdir statements in the else clause only happens if the directory was never there to begin with.

Try running the script twice in a row.